### PR TITLE
1389 | Home page HCM

### PR DESCRIPTION
**Pull request by chris-lawton**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1230241389)

### Description of Changes Made
- Adds visual indicator to primary links and blog posts in HCM
- Add hover/focus state to the same links in HCM
- Ensure the external link icon can be seen in light or dark mode in HCM

### How to Test
- Visit the homepage locally
- Enable `forced-colors: active` for HCM and `prefers-color-scheme` for light/dark mode

### Screenshots

<details>
  <summary>Expand to see more</summary>

Screenshot 2023-07-27 at 11 43 08

Screenshot 2023-07-27 at 11 43 16

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [ ] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [ ] Tests and linting passes.
- [ ] Consider updating documentation. If you don't, tell us why.
- [ ] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

**Comment by helenb** [comment]
There are actually some issues with the link focus state for non-HCM mode on the home page. I wonder if we should fix these together in case changes to the base focus state affect the HCM? The issue is for smaller monitors that get the mobile styles. The slide in effect for the underline on the menu links no longer happens. There's also no focus state for the 'more' button.

**Comment by helenb** [comment]
With 'prefers colour scheme: light' the logo is no longer visible.
With 'prefers colour scheme: dark' there is no visible indicator on the menu links when tabbing.

**Comment by helenb** [comment]
### Some issues from testing

**With 'prefers colour scheme:light'**
- Logo is not visible
- More button has no hover or focus state
- The hover state on the menu items is an extra underline, but the hover state on the blog items is that the underline disappears. I'm not sure how crucial this is - but I know RNIB were very keen that all hover states were consistent in HCM
- No hover state on footer links

**With 'prefers colour scheme:light'**
- Some logos in the footer are not visible
- No focus or hover state on the 'more' button
- Hover states on menu items and blog links are inconsistent - same as in light mode
- No hover state on footer links
- The social media links are hard to see

I've realised some of the above issues may be covered by the header and footer ticket

File: tbx/static_src/sass/abstracts/_mixins.scss [hunk 1]
```diff
@@ -172,3 +172,24 @@
...
+    }
+}
+
+/* ============================================
+    High Contrast mode (forced-colors: active)
+*/
+
+@mixin hcm() {
+    @media (forced-colors: active) {
+        @content;
```
Comment (helenb) [inline tbx/static_src/sass/abstracts/_mixins.scss:193]: TIL!

**Review by helenb — CHANGES_REQUESTED** [review]
Some fixes needed as mentioned in the comments

**Comment by chris-lawton** [comment]
Thanks @helenb 

Header and Footer points should be covered in https://github.com/torchbox/wagtail-torchbox/pull/510

I'll look at adding hover/focus state to the 'More' button.

I mentioned on the other MR but the site is very inconsistent with it's hover states in general (before viewing with any media query) across links so it makes it tricky deciding what to do for high contrast mode. I feel we'd need to have an audit which would then help inform what to do for HCM.

**Pull request merged**